lib/xcodeproj/workspace.rb

```diff
@@ -146,21 +217,36 @@ def root_xml(contents)
 <?xml version="1.0" encoding="UTF-8"?>
 <Workspace
    version = "1.0">
-   #{contents.strip}
+#{contents.rstrip}
 </Workspace>
       DOC
     end
 
-    # @return [String] The XML representation of a file reference.
+    # @return [String] The Xcode-specific XML formatting of an element start
```

Comment:
Convention is to put `@return` doc lines at the end after `@param` lines, not before